orchagent/aclorch.cpp

```diff
@@ -2001,6 +2002,12 @@ bool AclRulePacket::validateAddAction(string attr_name, string _attr_value)
             actionData.parameter.booldata = true;
             action_str = ACTION_DO_NOT_NAT_ACTION;
         }
+        // handle PACKET_ACTION_DISABLE_TRIM in ACTION_PACKET_ACTION
+        else if (attr_value == PACKET_ACTION_DISABLE_TRIM)
+        {
+            actionData.parameter.booldata = true;
+            action_str = ACTION_DISABLE_TRIM;
+        }
```

Comment:
@nazariig Do we have option to disable 'disable_trim' action as part of action update?

Reply:
@kperumalbfn once queue is configured as trimming eligible by attaching a buffer profile with a dedicated attribute, all packets forwarded to the queue are automatically considered as trimming eligible. In order to disable this behavior for certain types of flows based on a specific match criteria, ACL packet action `DISABLE_TRIM` is used. You can think of it like we include everything and then exclude partially what is needed. For configuration examples please refer to the HLD

Reply:
@nazariig I understand that queue is eligible for trimming and ACL action is used to disable the trimming for packets matching the ACL. Once ACL is configured with disable_trim, how do we enable trimming for the flow?

Reply:
@kperumalbfn currently, the only way to re-enable trimming for the flow is to remove the ACL rule which disables it